manifests/journald.pp

```diff
@@ -7,18 +7,13 @@
 
   assert_private()
 
-  if $systemd::journald_persist_log {
-    $journald_dir = 'directory'
-  } else {
-    $journald_dir = 'absent'
-  }
-
-  file { '/var/log/journal':
-    ensure => $journald_dir,
-    force  => true,
-    owner  => 'root',
-    group  => 'systemd-journal',
-    mode   => '2755',
+  if $systemd::journald_settings['Storage'] == 'auto' {
```

Comment:
Looking at the `journald` and `systemd` documentation, I would recommend not managing this space at all.

Systemd should "do the right thing" and this may interfere with that.

If there is a bug that needs to be worked around [like this one](https://github.com/systemd/systemd/issues/5264) then users should do that explicitly instead of having puppet fight with systemd.